packages/edit-site/src/components/layout/index.js

```diff
@@ -81,7 +81,7 @@ export default function Layout( { route } ) {
 	return (
 		<>
 			<CommandMenu />
-			<KeyboardShortcutsRegister />
+			{ canvas === 'view' && <KeyboardShortcutsRegister /> }
 			<KeyboardShortcutsGlobal />
```

Comment:
* IMO we can improve the naming. "Global" shouldn't be part of the labels, since we're always dealing with the same editor-wide shortcuts. It's more "define" vs. "activate", "register" vs. "enqueue", etc. No?

Reply:
For now I'll rename to `SaveKeyboardShortcut` and have a single file. If we need other site editor shortcuts let's create that folder then.

Reply:
The separation in rendering the two components looked so intentional that I read between the lines and assumed it was a performance concern (i.e. define the shortcut once, then toggle it more cheaply according to `view`). I should have not assumed and should have just asked. :)

The new setup with `SaveKeyboardShortcut` is much clearer, and I appreciate the [YAGNI](https://en.wikipedia.org/wiki/You_aren%27t_gonna_need_it) spirit.

Reply:
The intention was for the shortcut to always be available in the global "keyboard shortcuts" modal regardless of whether it was active or not.